### PR TITLE
copy to clipboard added in codeblock and color changed

### PR DESCRIPTION
**Pull request by jungsominh**
please take look in codeblock preview.

### CODE REVIEW

File: blog/DataScience/index.html [hunk 1]
```diff
@@ -137,6 +137,10 @@ <h2 class="rakibhhridoy-heading">DataFrame crash course</h2>
 												<pre class="language-python"><code>
 													import pandas as pd
 													import numpy as np
+
```
Comment (rakibhhridoy) [inline blog/DataScience/index.html:140]: no harmful code.

File: hstyle/blog/css/prism.css [hunk 2]
```diff
@@ -59,7 +59,7 @@ pre[class*=language-]{
     color:#e6db74
 }
 .token.keyword{
-    color:#ef66dd
+    color:#7dd634
```
Comment (rakibhhridoy) [inline hstyle/blog/css/prism.css:62]: nice color!

File: hstyle/blog/css/prism.css [hunk 3]
```diff
@@ -162,4 +162,54 @@ pre[class*=language-].line-numbers>code{
     padding-right:.8em;
     text-align:right
 }
+
```
Comment (rakibhhridoy) [inline hstyle/blog/css/prism.css:165]: please exclude unnecessary codes..

File: hstyle/blog/js/prism.js [hunk 4]
```diff
@@ -1,12 +1,1144 @@
...
+var _self = "undefined" != typeof window ? window : "undefined" != typeof WorkerGlobalScope && self instanceof WorkerGlobalScope ? self : {},
```
Comment (rakibhhridoy) [inline hstyle/blog/js/prism.js:3]: what have you changed? It looks similar..

**Review by rakibhhridoy — CHANGES_REQUESTED** [review]

File: hstyle/blog/css/prism.css (hunk 2 above)
Comment (jungsominh) [reply hstyle/blog/css/prism.css:62]: thanks.

File: hstyle/blog/css/prism.css (hunk 3 above)
Comment (jungsominh) [reply hstyle/blog/css/prism.css:165]: there's no code that is unnecessary.

File: hstyle/blog/js/prism.js (hunk 4 above)
Comment (jungsominh) [reply hstyle/blog/js/prism.js:3]: I've just formatted the js code to look easier. The code is from prism.js
the code is taken from:
https://prismjs.com/download.html#themes=prism-okaidia&plugins=toolbar+copy-to-clipboard
..

File: hstyle/blog/css/prism.css (hunk 3 above)
Comment (rakibhhridoy) [reply hstyle/blog/css/prism.css:165]: I see..

Comment (rakibhhridoy) [reply hstyle/blog/css/prism.css:165]: Ok, that looks good..

File: hstyle/blog/js/prism.js (hunk 4 above)
Comment (rakibhhridoy) [reply hstyle/blog/js/prism.js:3]: I've seen the code.. It looks ok

**Review by rakibhhridoy — APPROVED** [review]

**Pull request merged**